src/renderer/widgets/CreateWallet/ui/MultisigWallet/SelectMultisigWalletType.tsx

```diff
@@ -1,91 +1,140 @@
-    title: t('createMultisigAccount.singleChain.title'),
+  const flexibleMultisigOption = {
+    id: MultisigWalletType.FLEXIBLE,
+    value: MultisigWalletType.FLEXIBLE,
+    title: t('createMultisigAccount.flexibleMultisig.flexible'),
+    description: t('createMultisigAccount.selectMultisigDescription.flexibleDescription'),
+  };
+
+  const regularMultisigOption = {
+    id: MultisigWalletType.REGULAR,
+    value: MultisigWalletType.REGULAR,
+    title: t('createMultisigAccount.multisig'),
+    description: t('createMultisigAccount.selectMultisigDescription.regularDescription'),
   };
 
-  const multiChainOption = {
-    id: MultisigWalletType.MULTI_CHAIN,
-    value: MultisigWalletType.MULTI_CHAIN,
-    title: t('createMultisigAccount.multiChain.title'),
+  const handleContinue = () => {
+    selectMultisigModel.events.selectMultisigType(walletType!);
+
+    if (walletType === MultisigWalletType.FLEXIBLE) {
+      flexibleMultisigModel.events.stepChanged(Step.NAME_NETWORK);
+
+      return;
+    }
+    flowModel.events.stepChanged(Step.NAME_NETWORK);
   };
 
   return (
-    <BaseModal
-      title={<HeaderTitleText className="py-[3px]">{t('createMultisigAccount.title')}</HeaderTitleText>}
-      isOpen={isOpen}
-      panelClass="w-[664px]"
-      onClose={onClose}
-    >
-      <RadioGroup
-        className="mt-7 flex gap-6"
-        activeId={walletType}
-        options={[singleChainOption, multiChainOption]}
-        onChange={(option) => setWalletType(option.value)}
-      >
-        <RadioGroup.CardOption option={singleChainOption}>
-          <div className="flex flex-col gap-4 pl-3.5">
-            <BodyText className="list-item list-disc text-text-primary">
-              {t('createMultisigAccount.singleChain.featureOne')}
-            </BodyText>
-            <BodyText className="list-item list-disc text-text-primary">
-              {t('createMultisigAccount.singleChain.featureTwo')}
-            </BodyText>
-            <BodyText className="list-item list-disc text-text-primary">
-              {t('createMultisigAccount.singleChain.featureThree')}
+    <>
+      <Modal.Title close>
+        <HeaderTitleText>{t('createMultisigAccount.createMultisigWallet')}</HeaderTitleText>
+      </Modal.Title>
+      <Modal.Content>
+        <RadioGroup
+          className="mx-5 my-4 flex gap-x-6"
+          activeId={walletType}
+          options={[flexibleMultisigOption, regularMultisigOption]}
+          onChange={(option) => setWalletType(option.value)}
+        >
+          <RadioGroup.CardOption option={flexibleMultisigOption}>
+            <div className="flex flex-col gap-4">
+              {descriptionMultisig.map((item) => (
+                <div className="flex items-start gap-x-2" key={item.text}>
+                  <Icon name="checkmarkOutline" className="mt-1 shrink-0 text-text-positive" size={14} />
+                  <BodyText>
+                    <Trans t={t} i18nKey={item.text} />
```

Comment:
why not `t('...')`?

Reply:
Text for descriptions contains tags for bold text and with `t('...')` it will not be parced correctly